src/index.tsx

```diff
@@ -1,4 +1,9 @@
 import { createRoot } from 'react-dom/client';
 import { App } from './App';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
```suggestion
import { HashRouter } from 'react-router-dom';
```
use HashRouter instead BrawserRouter - it will help to solve issues with routing on gh pages.